cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -1528,7 +1528,11 @@ def _validate_spatial_assay_ontology_term_id(self):
 
     def _validate_spatial_cell_type_ontology_term_id(self):
         """
-        Validate cell type ontology term id is "unknown" if Visium, is_single is True and in_tissue is 0.
+        if dataset row obs.assay is Visium, uns.spatial.is_single is True, and obs.in_tissue is 0:
+
+        Validate cell type ontology term id is "unknown"
+        and organism cell type ontology term id is "unknown" (or "na", for when this field is not applicable
```

Comment:
i think it would be a bit more clear to have a different error message if you have a non-zebrafish/fly organism and this field is not `na`. maybe something like `For organism <organism name>, organism_cell_type_ontology_term_id must be 'na'`. Otherwise I feel like the error message is a bit confusing